docs/source/recommendations.rst

```diff
@@ -3,16 +3,25 @@
 Recommendations
 ===============
 
-Here we would like to list existing recommendations from various
-foundations and organizations
+List of existing recommendations from various foundations and organizations.
 
-International Coordination Facility (INCF) Neuroimaging taskforce
------------------------------------------------------------------
+Legal recommendations
+---------------------
 
-`Data Sharing Language Recommendations
-<http://datasharing.incf.org/ni/Data_Sharing_Language_Recommendations>`_
-(J.Turner, D.Kennedy, JB Poline, J. Roberts) provides a thorough
-summary over current state of used consent forms among neuroimaging
-studies and some data sharing initiatives.  It further develops a set
-of recommendations on consent form language for perspective studies
-and analysis of public sharing of legacy data.
+Canada: `NCEHR <http://www.ncehr-cnerh.org/>`_ is a national organization
+mandated to advance the protection and well-being of human participants in
+research and to foster high ethical standards for the conduct of research
+involving humans.
+EU: The General Data Protection Regulation `(GDPR <https://gdpr.eu/>)`_ governs
+how any data (i.e. including research data) should be managed.
+USA: `The Office for Human Research Protections (OHRP <https://www.hhs.gov/ohrp/>)`_
+provides leadership in the protection of the rights, welfare, and wellbeing of
+human subjects involved in research conducted or supported by the U.S.
+Department of Health and Human Services (HHS).
```

Comment:
make it into an itemized list? otherwise it would all be reformatted into a single paragraph

Reply:
I will push this change to also demo our new feature -- RTD test builds in PRs ;)

Reply:
not sure what this implies for rst? just #.

Reply:
yes indeed, thx